lib/perl/Genome/Model/Tools/Speedseq/Sv.pm

```diff
@@ -0,0 +1,137 @@
+package Genome::Model::Tools::Speedseq::Sv;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Model::Tools::Speedseq::Sv {
+	is => 'Genome::Model::Tools::Speedseq::Base',
+	has_param => [
+
+		full_bam_file => {
+			is => 'Text',
+			doc => 'full BAM file(s) (comma separated list)',
+			example_values => ['input1.bam,input2.bam'],			
+			tool_param_name => 'B',
+			tool_input_file => 1,
+		},
+		split_read_bam_file => {
+                        is => 'Text',
+                        doc => 'split reads BAM file(s) (comma separated list)(Auto Generated if absent)',
+                        example_values => ['input1.splitters.bam,input2.splitters.bam'],
+                        tool_param_name => 'S',
+			is_optional => 1,
+                        tool_input_file => 1,
+                },
+		discordant_read_bam_file => {
+                        is => 'Text',
+                        doc => 'discordant reads BAM file(s) (comma separated list) (Auto Generated if absent)',
+                        example_values => ['input1.bam'],
+                        tool_param_name => 'D',
+                        is_optional => 1,
+                        tool_input_file => 1,
+                },
+		reference_fasta => {
+			is => 'Text',
+			doc => 'fasta file (indexed with bwa)',
+			example_values => ['refernce.fa'],
+			tool_input_file => 1,
+			tool_param_name => 'R',
+		},
+        	output_prefix => {
+        		is => 'Text',
+        		doc => 'output prefix [in1.bam]',
+        		is_optional => 1,
+        		example_values => ['in1.fq','in.realign'],
+        		tool_param_name => 'o',
+        	},
+		threads => {
+        		is => 'Integer',
+        		doc => 'threads [1]',
```

Comment:
It's not necessary for the `doc` to contain the `default_value`.  The output for `--help` will list it in a structured way.